templates/auto_master.erb

```diff
@@ -0,0 +1,8 @@
+#
+## This file is maintained by puppet. Manual changes will be overwritten.
+#
+
+<% if @mounts -%>
+<% @mounts.each do | mount| -%>
```

Comment:
The array should be sorted, else it could come out different between puppet runs with ruby 1.8.7 and cause churn.